src/twisted/web/newsfragments/.gitignore

```diff
@@ -0,0 +1 @@
+!.gitignore
```

Comment:
`git` is terrible, but this is an idiom.  Even if it's a dumb idiom I don't want Twisted to have an exception, especially for something that's supposed to make contributing easier for newcomers.

The file stays in the picture.